docs/units/structured_units.rst

```diff
@@ -0,0 +1,130 @@
+.. _structured_units:
+
+Structured Units
+****************
+
+Numpy arrays can be :doc:`structured arrays <numpy:user/basics.rec>`, where
+each elements consists of multiple fields. These can be used with |Quantity|
+using a |StructuredUnit|, which provides a |Unit| for each field. For example,
+this allows the PyERFA_ wrappers around the ERFA_ routines to support having
+units on the position-velocity functions.
```

Comment:
It feels like the "for example" here might be too "inside baseball" :). Maybe just generalize it to something like:
```suggestion
using a |StructuredUnit|, which provides a |Unit| for each field. For example,
this allows constructing a single |Quantity| object with position and velocity 
fields that have different units, but are contained within the same object.
```

Reply:
Done (while still keeping a reference to ERFA)